### DISCUSSION TITLE
Vertico error in post-hook

### DISCUSSION THREAD

**Discussion by plk** [minad/vertico#420] [Q&A]
When vertico tries to start, I get:

```
[Error in post-command-hook (vertico--exhibit): (void-variable aff)]
```
Stack trace (using the (advice) hook to get a real trace) doesn't seem to tell me much more than that. Is there something obvious I'm missing here?

**Comment by minad** [comment]
Hi! This is likely an installation problem. Can you please go through the steps described at https://github.com/minad/consult#bug-reports, only the Vertico-related ones of course?

**Comment by plk** [comment]
Problem seems to be with the "+icons" option in doom emacs. Removing this makes vertico work, but without the marginalia icons.

**Comment by plk** [comment]
Apologies for the noise. I had some broken, duplicate `nerd-icons-completion.el` in my path and this was the issue.
